sdk/tracing/azotel/otel_test.go

```diff
@@ -0,0 +1,259 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azotel
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	azruntime "github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
+	"github.com/Azure/azure-sdk-for-go/sdk/tracing/azotel/internal"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func TestNewTracingProvider(t *testing.T) {
+	exporter := &testExporter{}
+
+	otelTP := tracesdk.NewTracerProvider(tracesdk.WithBatcher(exporter))
+
+	options := azcore.ClientOptions{}
```

Comment:
I briefly wondered - did Joel do this because using a literal initialize would actually use one _extra_ line? 

And I have to say, I couldn't know for sure.

Reply:
Turns out, using a literal was one line shorter than what I had.  Save the new-lines!